### PR TITLE
Fixed error running on plugin OWTF-CM-001

### LINKED ISSUE #1076: Error on running some plugins
## Expected Behavior

1) OWTF-CM-001
```
`owtf_1  | [-] Aborted by Framework: Error when calling 'codecs_open'! [Errno 2] No such file or directory: 'owtf_review/targets/https__www.youtube.com/partial/Testing For Ssl-Tls/active/TLSSLed_v1.2.sh (openssl/sslscan).txt'.
`
```
Report is not being generated on ActiveSSLCmds.

2)OWTF-PTES-009
`python /usr/share/doc/python-impacket-doc/examples/samrdump.py` Not found

## Steps to Reproduce (for bugs)

1.RUN plugin OWTF-CM-001 and OWTF-PTES-009

### PR DESCRIPTION
**Pull request by amanpro30**
codecs.open() was not able to create file as the filename was containing "/", "(" and ")". So, I have modified the multi_replace() in utils/strings.py to change these symbols to "_" and "".

## Description
```
[+] __________ 1 - Target: https://youtube.com -> Plugin: Testing For Ssl-Tls (web/active) __________
[+] 
[+] Executing :

cd owtf_review/targets/https__youtube.com/partial/Testing\ For\ Ssl-Tls/active;sh /home/amanpro30/Desktop/GSoC/owtf/owtf/scripts/ssl/run_tlssled.sh /usr/bin/tlssled youtube.com 443


[+] ------> Execution Start Date/Time: 23/03/2021-01:05
[+] 
/usr/lib/python3.8/subprocess.py:844: RuntimeWarning: line buffering (buffering=1) isn't supported in binary mode, the default buffer size will be used
  self.stdout = io.open(c2pread, 'rb', bufsize)
[+] 
[+] [*] SSL Handshake Check OK: The host youtube.com appears to speak SSL/TLS on port: 443
[+] 
[+] /home/amanpro30/Desktop/GSoC/owtf/owtf/scripts/ssl/run_tlssled.sh: 50: /usr/bin/tlssled: not found
[+] Time=0h,  0s,  19ms
[-] Aborted by Framework: Error when calling 'codecs_open'! [Errno 2] No such file or directory: 'owtf_review/targets/https__youtube.com/partial/Testing For Ssl-Tls/active/TLSSLed_v1.2.sh (openssl/sslscan).txt'.
```

Corrected:
```
[+] __________ 2 - Target: https://youtube.com -> Plugin: Testing For Ssl-Tls (web/active) __________
[+] 
[+] Executing :

cd owtf_review/targets/https__youtube.com/partial/Testing_For_Ssl-Tls/active;sh /home/amanpro30/Desktop/GSoC/owtf/owtf/scripts/ssl/verify_ssl_cipher_check.sh /root/.owtf/tools/restricted/ssl/ssl-cipher-check/ssl-cipher-check.pl youtube.com 443


[+] ------> Execution Start Date/Time: 23/03/2021-00:59
[+] 
/usr/lib/python3.8/subprocess.py:844: RuntimeWarning: line buffering (buffering=1) isn't supported in binary mode, the default buffer size will be used
  self.stdout = io.open(c2pread, 'rb', bufsize)
[+] ------------------------------------------------------
[+] /home/amanpro30/Desktop/GSoC/owtf/owtf/scripts/ssl/verify_ssl_cipher_check.sh - (0.1) based on ssl-cipher-check.pl
[+] Author: Abraham Aranguren @7a_ http://7-a.org
[+] ------------------------------------------------------
[+] 
[+] 
[+] [*] SSL Handshake Check OK: The host youtube.com appears to speak SSL/TLS on port: 443
[+] 
[+] [*] Analyzing SSL/TLS on youtube.com:443 ...
[+] [*] Step 1 - sslcan-based analysis
[+] 
[+] [*] ssl-cipher-check-based analysis (for comparison/assurance purposes)
[+] [*] NOTE: If you get errors below, try running: "apt-get install gnutls-bin"
[+] 
[+] [*] Running ssl-cipher-check.pl on youtube.com:443...
[+] 
[+] [*] Testing for SSLv2 ...
[+] 
[+] [*] Testing for NULL cipher ...
[+] 
[+] [*] Testing weak ciphers ...
[+] 
[+] [*] Testing strong ciphers ...
[+] 
[+] [*] Default cipher: ...
[+] 
[+] [*] New files created:
[+] -rw-r--r-- 1 root root 164 Mar 23 00:59 ssl_cipher_check_2021-03-23_00:59:46.err
[+] -rw-r--r-- 1 root root   0 Mar 23 00:59 ssl_cipher_check_2021-03-23_00:59:46.log
[+] 
[+] 
[+] [*] done
[+] 
[+] Time=0h,  0s,  74ms
[+] 
[+] Executing :

cd owtf_review/targets/https__youtube.com/partial/Testing_For_Ssl-Tls/active;/root/.owtf/tools/restricted/ssllabs-scan/ssllabs-scan https://youtube.com > SSL_TLS_testing_functionality_from_SSLLabs.json; /home/amanpro30/Desktop/GSoC/owtf/owtf/scripts/ssl/reporter.py SSL_TLS_testing_functionality_from_SSLLabs.json


[+] ------> Execution Start Date/Time: 23/03/2021-00:59
[+] 
[+] /bin/sh: 1: /root/.owtf/tools/restricted/ssllabs-scan/ssllabs-scan: not found
[+] Traceback (most recent call last):
[+] File "/home/amanpro30/Desktop/GSoC/owtf/owtf/scripts/ssl/reporter.py", line 11, in <module>
[+] from owtf.utils.file import FileOperations
[+] ImportError: No module named owtf.utils.file
[+] Time=0s,  67ms
[+] 
[+] Executing :

cd owtf_review/targets/https__youtube.com/partial/Testing_For_Ssl-Tls/active;/usr/bin/o-saft +cipher --force-openssl youtube.com --port 443


[+] ------> Execution Start Date/Time: 23/03/2021-00:59
[+] 
[+] /bin/sh: 1: /usr/bin/o-saft: not found
[+] Time=0s,  28ms
[+] 
[+] Executing :

cd owtf_review/targets/https__youtube.com/partial/Testing_For_Ssl-Tls/active;sh /home/amanpro30/Desktop/GSoC/owtf/owtf/scripts/ssl/run_tlssled.sh /usr/bin/tlssled youtube.com 443


[+] ------> Execution Start Date/Time: 23/03/2021-00:59
[+] 
[+] 
[+] [*] SSL Handshake Check OK: The host youtube.com appears to speak SSL/TLS on port: 443
[+] 
[+] /home/amanpro30/Desktop/GSoC/owtf/owtf/scripts/ssl/run_tlssled.sh: 50: /usr/bin/tlssled: not found
[+] Time=0h,  0s,  44ms

```
## Related Issue
Fixed #1076 

## Reviewers
@viyatb @sharmamohit123 

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Other

### Checklist:

- [x] My code follows the code style (modified PEP8) of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by viyatb** [comment]
Can you add tests for this?

File: owtf/utils/strings.py [hunk 1]
```diff
@@ -39,7 +39,7 @@ def str2bool(string):
     return not (string in ["False", "false", 0, "0"])
 
 
-def multi_replace(text, replace_dict):
+def multi_replace(text, replace_dict, simple_text = False):
```
Comment (viyatb) [inline owtf/utils/strings.py]: `simple_text=False` (remove whitespace)

**Comment by viyatb** [comment]
can you rebase @amanpro30? I can merge after that

File: owtf/utils/strings.py (hunk 1 above)
Comment (amanpro30) [reply owtf/utils/strings.py]: Changed

**Pull request merged**